lib/tests.js

```diff
@@ -228,6 +228,25 @@ tests.androidLongTest = async function (driver, caps) {
   }
 };
 
+tests.androidLoadTest = async function (driver, caps) {
+  let testSeconds = 600;
+  let args = await driver.elementById('textOptions');
+  let goBtn = await driver.elementById('go');
+
+  await args.clear();
+  await args.sendKeys("-M 500 -v 20" + " -s " + testSeconds);
+
+  for (let testMinutes = testSeconds / 60; testMinutes > 0; testMinutes--) {
+    // Only the first click triggers the test, the rest of the clicks are ignored
```

Comment:
i see what you're trying to do here now; i think we can simplify this a bit (and don't forget template strings):

```js
const iterations = 20; // run for 10 minutes
const intervalMs = 30000;
...
await args.sendKeys(`-M 500 -v 20 -s ${Math.floor(iterations * intervalMs / 1000)}`);

for (let i = 0; i < iterations; i++) {
  // comment
  await goBtn.click();
  await driver.sleep(intervalMs);
}
```

Reply:
Thanks that looks cleaner